pages/doc/logging_logs_chart.md

```diff
@@ -0,0 +1,133 @@
+    </li>
+  </ul>
+  {{site.data.alerts.end}}
+  </p>
+    </td>
+  <td width="60%">
+    <img src="images/logs_histogram_functions.png" alt="The Count drop-down menu."/>
+  </td>
+</tr>
+</table>
+
+I you select the stacked-column chart, the resulting chart shows the number of logs matching the selected filters distributed over the [selected time window](ui_charts.html#set-the-time-window-on-a-chart). The logs are grouped by the values of the selected grouping tags. You can zoom in on smaller time windows. You can also use the logs table chart to see the logs on your chart.
+  ![Logs histogram chart](images/logs_histogram_chart.png)
+
+## Logs Table Chart
+
+To troubleshoot faster, you can have the logs data on the same dashboard that you use to monitor your application, service, sources, etc., by using the logs table chart.
+
+<a id="create_logs_table_chart">
+<p><span style="font-size: medium; font-weight: 600">Create a Logs Table Chart</span></p>
+
+Follow these steps:
+
+1. Select **Dashboards** > **Create Chart** on the toolbar.
+1. Select the **Logs Table** chart type.
+1. From the **Data** drop-down menu, select **Logs (Beta)** and add the filters that you want.
+    {% include note.html content="The Count function is disabled when you select the logs table chart because you don’t need to group your logs in the table view." %}
+14. Click the **Format** tab to customize the data displayed on the chart.
+    <table>
+    <tbody>
+    <thead>
+    <tr><th width="20%">Option</th><th width="80%">Description</th></tr>
+    </thead>
+    <tr>
+      <td>
+        Sort by Timestamp
+      </td>
+      <td markdown="span">
+        By default, you see the latest logs on the top of the table (**Descending**). If you want to see the oldest logs, select **Ascending**.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Number of Message Lines
+      </td>
+      <td>
+        Select the number of log lines you want to see in the Message column to get a better preview of the log message.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        Columns to Show in Table
+      </td>
+      <td markdown="span">
+        Each log has required attributes, standard attributes, and custom tags. We tokenize the values of these attributes and tags so that you can filter and search logs. For more details, see [Log Attributes](logging_overview.html#log-attributes). Select the tags for which you want to see  see information in new columns.
+      </td>
+    </tr>
+    </tbody>
+    </table>
+1. On the top right, click **Save**.
+1. Specify the dashboard in which you want to insert the chart or create a new dashboard.
+
+<a id="view_logs_on_the_logs_table_chart">
+<p><span style="font-size: medium; font-weight: 600">View Logs on the Chart</span></p>
+
+Once the chart is in a dashboard, you can: 
+
+![An annotated screenshot of the logs table chart.](images/logging_logs_chart_features.png)
```

Comment:
An annotated screenshot of the logs table chart. The annotations are listed as bullets below.